proto/decentraland/sdk/components/video_event.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package decentraland.sdk.components;
+
+import "decentraland/sdk/components/common/id.proto";
+option (common.ecs_component_id) = 1044;
+
+message PBVideoEvent {
+  uint32 timestamp = 1; // monotonic counter
```

Comment:
I'd also add the tick_number to detect if the event was triggered on the same frame. Can copy from raycast result